components/timeline/TimelineDiscover.vue

```diff
@@ -6,10 +6,54 @@ const { locale: lang } = useI18n()
 const locale = getLanguageForRecs(lang.value)
 
 const recommendations: Recommendation[] = await $fetch(`/api/recommendations?locale=${locale}`)
+
+// Shorten a string to less than maxLen characters without truncating words.
+function shorten(str: string, maxLen: number): string {
+  if (str.length <= maxLen)
+    return str
+  return `${str.slice(0, str.lastIndexOf(' ', maxLen))}...`
+}
+
+// This is temporary untill we are able to fetch from the new endpoint
+function updateUTM(url: string): string {
+  return url.replace('pocket-newtab', 'mozilla')
+}
```

Comment:
The temporary API returns a utm param, but the new one won't, so this function will go away in favor of string concatenation when we declare the url in the `NuxtLink`